lib/IdpListsServiceCsv.php

```diff
@@ -184,4 +195,10 @@ private function arrayToIdp($csv) {
 		return $idp;
 	}
 
+	private function commit($file, $entityID){
+		$idpListsDir = dirname($file);
+		$fileName = basename($file);
+		shell_exec("cd $idpListsDir && pwd && git add $file && git commit -m\"Updated $fileName  \n\n * Added record with entityId: $entityID to $fileName\"");
```

Comment:
- why is there a 'pwd' command?
- before you pass any variable to the shell_exec escape all dangerous characters, see http://php.net/manual/en/function.escapeshellarg.php
- it is better to use exec() instead of shell_exec() because in exec() you can get return code. You should check the return code and if there is some problem, then log it
- also provide --author="Automated commit <no-reply@no.org>"